apps/client/src/widgets/report/components/maturity/maturity.tsx

```diff
@@ -0,0 +1,22 @@
+import * as styles from './maturity.css';
+
+interface MaturityProps {
+  age: number; // api 연동
+}
+
+const MATURITY = '만기';
+const AGE = '세';
```

Comment:
```suggestion
const AGELABEL = '세';
```

은 어떠신가요? 사용될때 소문자 age 와 AGE 가 같이 사용되니깐 조금 혼동이 있는 것 같아요.